src/components/SocialIcons/SocialIcons.component.tsx

```diff
@@ -0,0 +1,14 @@
+import facebookImg from '@/assets/img/footerImg/facebook.png';
+import instagramImg from '@/assets/img/footerImg/instagram.png';
+import linkedInImg from '@/assets/img/footerImg/linkedIn.png';
+import { CreateSocialIcon } from '@/components/CreateSocialIcon/CreateSocialIcon.tsx';
+
+import styles from './socialIcons.module.css';
+
+export const SocialIcon = () => (
+    <ul className={styles.socialNetwork}>
+        <CreateSocialIcon img={facebookImg} url={'https://www.facebook.com/profile.php?id=100015349961940'} altName={'Facebook'} />
```

Comment:
like for me better save all data in array and use `map` for cycle. Because now you duplicated code. If you will have for example 15 icons it will be bad to copy past